test/release/examples/users-guide/base/configs3-setops.good

```diff
@@ -0,0 +1,2 @@
+10
+1e-09
```

Comment:
I don't think this file is referenced anywhere.  Is it just an extra copy of configs3-seteps.good?